src/main/java/edu/harvard/iq/dataverse/ManageFilePermissionsPage.java

```diff
@@ -115,6 +115,17 @@ public void setDataset(Dataset dataset) {
     }
     
     
+    private boolean backingShowDeleted = true;
+
+    public String showDeletedCheckboxChange() {
```

Comment:
@sekmiller Only small comment, does this need to return "" or can it just be void?

Reply:
you're right. thanks. I made the change.